lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+
+  def initialize; end
+
+  def create_vehicles(database)
+    database.map do |vehicle|
+      Vehicle.new({
+                  vin: vehicle[:vin_1_10],
+                  year: vehicle[:model_year].to_i,
+                  make: vehicle[:make],
+                  model: vehicle[:model],
+                  engine: :ev
+                })
```

Comment:
This closing bracket and paren should be in alignment with the `V` in `Vehicle` on line 7

Reply:
Understood.